policy-addon-kubernetes/helm-charts/azure-policy-addon-aks-engine/README.md

```diff
@@ -0,0 +1,10 @@
+# Azure Policy Add-on for AKS Engine
```

Comment:
It seems weird to have this README 3 levels deep.  What is the "policy-addon-kubernetes" folder for?  Will there be other "helm-charts"?  

From someone who doesn't know what any of this is there isn't enough top-level documentation describing what the purpose of this folder tree is and how it should be used.